pkg/config/config.go

```diff
@@ -172,8 +173,9 @@ type (
 	}
 
 	seed struct {
-		Enabled  bool     `toml:"enabled"`
-		SqlPaths []string `toml:"sql_paths"`
+		Enabled      bool     `toml:"enabled"`
+		GlobPatterns []string `toml:"sql_paths"`
+		SqlPaths     []string `toml:"-"`
```

Comment:
**praise**

I really need to remember this trick to add stuff in the config structure that are not in the toml.